lib/nautic_net/serial/six_fab/server.ex

```diff
@@ -0,0 +1,59 @@
+defmodule NauticNet.Serial.SixFab.Server do
```

Comment:
This module is unused.

Reply:
Yes. This is something I wanted to discuss. I believe I followed the pattern that `pican` has... which also seems to have one of the modules unused. Is that cruft as well or did you have plans that never materialized?

Reply:
`NauticNet.CAN.PiCAN.Server` is started explicitly by `NauticNet.CAN.PiCAN.Driver` because I needed a separate GenServer to handle the implementation details of interfacing with the `socketcand` process. In this case, the driver doesn't have its own state and it doesn't need to spawn another GenServer (the UART callbacks are in `NauticNet.Serial.Server`), so this moudle isn't needed.

Reply:
Removed after discussing last week.